api/src/main/java/org/openmrs/module/shr/cdahandler/api/db/CdaImportServiceDAO.java

```diff
@@ -2,13 +2,7 @@
 
 import java.util.List;
 
-import org.openmrs.Concept;
-import org.openmrs.ConceptMapType;
-import org.openmrs.ConceptReferenceTerm;
-import org.openmrs.ConceptSource;
-import org.openmrs.Obs;
-import org.openmrs.Order;
-import org.openmrs.activelist.ActiveListItem;
+import org.openmrs.*;
```

Comment:
Avoid a wildcard imports. You can set this in IntelliJ in File -> Settings -> Editor -> Code Style -> Java. There you can check a _Use single class import_ and increase a values in fields _Class count to use import with '*'_ and _Names count to use static import with '*'_ to some big value e.g. 50.

Additionally you can set an optimization of imports in File -> Settings -> Editor -> General -> Auto Import.